docs/user-event/intro.mdx

```diff
@@ -42,7 +42,13 @@ factors in that the browser e.g. wouldn't let a user click a hidden element or
 type in a disabled text box.  
 This is
 [why you should use `user-event`](https://ph-fritsche.github.io/blog/post/why-userevent)
-to test interaction with your components.
+to test interaction with your components. 
+
+There are, however, some user interactions or aspects of these [that aren't yet implemented and thus can't yet be described with `user-event`](https://github.com/testing-library/user-event/issues?q=is%3Aopen+label%3Aaccuracy%2Cenhancement).
+In these cases you can use `fireEvent` to dispatch the concrete events that your software relies on.
+
+Note that this makes your component and/or test reliant upon your assumptions about the concrete aspects of the interaction being correct.
+Therefore if you already put in the work to specify the correct aspects of such interaction, please consider contributing to this project so that user-event might cover these cases too.
```

Comment:
```suggestion
Therefore if you already put in the work to specify the correct aspects of such interaction, please consider contributing to this project so that `user-event` might cover these cases too.
```